app/src/main/java/de/tum/in/tumcampus/models/managers/MockMoodleManager.java

```diff
@@ -0,0 +1,154 @@
+package de.tum.in.tumcampus.models.managers;
+
+import android.content.Context;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+import de.tum.in.tumcampus.models.MoodleCourse;
+import de.tum.in.tumcampus.models.MoodleEvent;
+import de.tum.in.tumcampus.models.MoodleEventsList;
+import de.tum.in.tumcampus.models.MoodleToken;
+import de.tum.in.tumcampus.models.MoodleUser;
+import de.tum.in.tumcampus.models.MoodleUserCourseList;
+
+/**
+ * Created by a2k on 6/7/2015.
+ * This class is a stub for RealMoodleManager
+ */
+public class MockMoodleManager extends MoodleManager{
```

Comment:
Totally forgot about it, we used it when we started developing. It can be removed.